lib/ical/binary.js

```diff
@@ -1,181 +1,174 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
- * Portions Copyright (C) Philipp Kewisch, 2011-2015 */
-
+ * Portions Copyright (C) Philipp Kewisch */
 
 /**
- * This symbol is further described later on
- * @ignore
+ * Represents the BINARY value type, which contains extra methods for encoding and decoding.
+ *
+ * @class
+ * @alias ICAL.Binary
  */
-ICAL.Binary = (function() {
+class Binary {
```

Comment:
This can be changed to `export default class {`, skipping the name Binary and skipping the last line of the file (export default Binary).

Likewise in the other files.

Reply:
Having a named class (instead of an anonymous export) is better for:

* Generating docs
* Logging
* Editor auto-completion / “IntelliSense”, etc.

Reply:
`export default class Binary {` would also work.

Reply:
👍

Reply:
I can't quite put my finger on it now, but I believe I had a specific reason for this. I believe it was connected to the "export default function' is another special case" section of article: https://jakearchibald.com/2021/export-default-thing-vs-thing-as-default/  and/or because there was a situation where it made sense to do it in the one or other file, and then it made sense to keep it that way for consistency. To get this in I'm going to keep it this way, but if someone wants to go through the exports and change them all, and it still works afterwords, I'm happy to take a PR.